recipes/django-leaflet/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "django-leaflet" %}
+{% set version = "0.22.0" %}
+{% set sha256 = "2a1f4f31bff2b3a58158372f7910053670483e3e11f8ac365ce75f3e3a4ac11c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ordereddict  # [py<27]
```

Comment:
I'd remove this as we only support py>=27

Reply:
OK :) thanks

Reply:
The rationale here is beyond what we, conda-forge, support. It is OK to add pre-selectors to what we do not support in order to make the recipes generic to others, but `<27` is really frowned upon these days (for some even <py34), so I agree that removing is best.